config/20.1/make.conf

```diff
@@ -30,11 +30,12 @@ OPTIONS_UNSET=		DOCS EXAMPLES GCC GSSAPI_BASE NLS OPENGL WAYLAND X11
 DEFAULT_VERSIONS=	python=${PRODUCT_PYTHON3:C/^./&./}
 DEFAULT_VERSIONS+=	python3=${PRODUCT_PYTHON3:C/^./&./}
 DEFAULT_VERSIONS+=	ruby=${PRODUCT_RUBY:C/^./&./}
+# XXX need to add if product is arm/arm64, go to openssl 1.0.2
 .if ${PRODUCT_FLAVOUR:tl} != "libressl"
```

Comment:
@fitchner I need help with this, I'm not familiar with makeconf to be able to make it work. Tried with `.if ${PRODUCT_FLAVOUR:tl} != "libressl" && ${PRODUCT_TARGET} != "arm" && ${PRODUCT_TARGET} != "arm64"` but returns malformed conditional.

Reply:
just set FLAVOUR=LibreSSL when building

Reply:
Noted. Will try that on next build.